src/components/interaction-designer/resource-editors/ResourceEditor.vue

```diff
@@ -159,6 +161,16 @@ export class ResourceEditor extends mixins(FlowUploader, Permissions, Routes, La
 
   SupportedContentType = SupportedContentType
 
+  iconsMap = new Map<string, string>()
+
+  created() {
+    this.iconsMap.set(SupportedMode.SMS, 'message')
+    this.iconsMap.set(SupportedMode.USSD, 'ussd')
+    this.iconsMap.set(SupportedMode.IVR, 'audio')
+    this.iconsMap.set(SupportedMode.RICH_MESSAGING, 'rich-messaging')
+    this.iconsMap.set(SupportedMode.OFFLINE, 'phone')
```

Comment:
FYI, we could initialize iconsMap with its values if you want to couple these directly to a single reference site: 
```
iconsMap = new Map<string, string>([
  [SupportedMode.SMS, 'message'],
  [SupportedMode.USSD, 'ussd'],
  ...
])
```

Reply:
I think this way is better. I changed it!